components/Start.js

```diff
@@ -108,6 +115,7 @@ const styles = StyleSheet.create({
     color: "#75708380",
   },
   controls: {
+    // justifyContent: "space-around",
```

Comment:
The white box can stay the same if you add a `minHeight` attribute. 300 looks fine for me.